### PR TITLE
Fix #793 Show error on admin page for account taken, instead of raised exception

### LINKED ISSUE #793: An ActiveRecord::RecordInvalid occurred in users#create:
This should not be an exception but instead an error that flows up to the admin form that caused it (probably `create` plus `transfer_errors_from` instead of `create!`)

An ActiveRecord::RecordInvalid occurred in users#create:

 Validation failed: Account has already been taken
 lib/entity.rb:44:in `block (2 levels) in exposes'

See email

### PR DESCRIPTION
**Pull request by joemsak**
See #793

### CODE REVIEW

**Comment by jpslav** [comment]
Looks good.  Merge when CI passes.

File: app/subsystems/user/create_user.rb [hunk 1]
```diff
@@ -20,11 +20,18 @@ def exec(account_id: nil, exchange_identifiers: nil,
...
         exchange_write_identifier: (exchange_identifiers || new_identifiers).write,
         account_id: account_id
       )
+
+      if user.to_model.valid?
+        outputs.user = user
+      else
+        fatal_error(code: :could_not_create_user,
+                    message: user.to_model.errors.first.join(' ').capitalize)
+      end
```
Comment (jpslav) [inline app/subsystems/user/create_user.rb]: weren't you considering calling `transfer_errors_from` instead of this block of code?

Comment (joemsak) [inline app/subsystems/user/create_user.rb]: ah okay have to use `user.to_model` but yes

**Comment by joemsak** [comment]
@jpslav awaiting CI

**Pull request merged**